include/libdnf/rpm/package_set.hpp

```diff
@@ -21,28 +21,29 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 #ifndef LIBDNF_RPM_PACKAGE_SET_HPP
 #define LIBDNF_RPM_PACKAGE_SET_HPP
 
+//namespace libdnf::rpm {
+//    class Package;
+//}
 
 #include "package.hpp"
+#include "package_set_iterator.hpp"
```

Comment:
This commit seems to be doing more than adding the iterator, it moves a lot of stuff around, doesn't it? I'm not asking to split it now necessarily, but at least specify it better in the commit message?